src/enveloping/CommitmentValidator.ts

```diff
@@ -0,0 +1,41 @@
+import log from 'loglevel'
+import { Commitment, CommitmentReceipt } from './Commitment'
+import { ethers } from 'ethers'
+
+/**
+ * The CommitmentValidator can verify a CommitmentReceipt signature
+ */
+export class CommitmentValidator {
```

Comment:
we should discuss whether this is behavior that could be responsibility of the CommitmentReceipt, or whether that would bloat it